parser/nodetype_string.go

```diff
@@ -0,0 +1,16 @@
+// generated by stringer -type=NodeType; DO NOT EDIT
```

Comment:
Are you supposed to check this in if it's being generated automatically? I haven't actually ever used `go generate`.